common/ajax-fetch/ajax-fetch-fr.html

```diff
@@ -0,0 +1,69 @@
+---
+{
+	"title": "Récupération de ressource via Ajax",
+	"language": "fr",
+	"altLangPage": "ajax-fetch-en.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-fr.html" }
+	],
+	"dateModified": "2023-08-25"
+}
+---
+<div class="wb-prettify all-pre"></div>
+<h2>Succès de récupération d'une ressource ajax</h2>
+<div data-ajax-replace="ajax/data-ajax-extra-fr.html">
+	<section>
+		<h3>Texte à être remplacé</h3>
+		<p>Ce texte sera remplacé.</p>
+	</section>
+</div>
+
+<h2>Échec de récupération d'une ressource ajax</h2>
+<div data-ajax-replace="failedUrl">
+	<section>
+		<h3>Texte à être remplacé</h3>
+		<p>Ce texte ne sera <strong>pas</strong> remplacé.</p>
+	</section>
+</div>
+
+<div class="row">
```

Comment:
We would just need an heading level 2

```suggestion
<h2>Code</h2>
<div class="row">
```